### PR TITLE
Add HTS Precompile tests for token check methods

### LINKED ISSUE #411: Add htsPrecompile acceptance tests support for token check methods
### Problem

No htsPrecompile acceptance tests support for token check methods exists

### Solution

Add support for the token verifications:

- `isToken(address _token) returns (uint64 statusCode, bool isToken)`
- `getTokenType(address _token) returns (uint64 statusCode, int32 tokenType)`

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by natanasow**
Signed-off-by: nikolay <n.atanasow94@gmail.com>

**Description**:

Add support for the token verifications:

    isToken(address _token) returns (uint64 statusCode, bool isToken)
    getTokenType(address _token) returns (uint64 statusCode, int32 tokenType)

**Related issue(s)**:

Fixes #411 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/server/tests/acceptance/htsPrecompile.spec.ts [hunk 1]
```diff
@@ -604,4 +604,57 @@ describe('HTS Precompile Acceptance Tests', async function () {
...
+      const tx = await baseHTSContract.getTokenTypePublic(HTSTokenContractAddress, { gasLimit: 1000000 });
+      const txReceipt = await tx.wait();
+
+      const responseCode = txReceipt.events.filter(e => e.event === 'ResponseCode')[0].args.responseCode;
+      expect(responseCode).to.equal(TX_SUCCESS_CODE);
+
+      const tokenType = txReceipt.events.filter(e => e.event === 'TokenType')[0].args.tokenType;
+      expect(tokenType).to.equal(0);
+    });
+    it('should returns 1 for getTokenType with passed HTS nft token', async function() {
```
Comment (Nana-EC) [inline packages/server/tests/acceptance/htsPrecompile.spec.ts:639]:
```suggestion
    it('should return 1 for getTokenType with passed HTS nft token', async function() {
```

File: packages/server/tests/acceptance/htsPrecompile.spec.ts [hunk 2]
```diff
@@ -604,4 +604,57 @@ describe('HTS Precompile Acceptance Tests', async function () {
...
+      const tx = await baseHTSContract.isTokenPublic(HTSTokenContractAddress, { gasLimit: 1000000 });
+      const txReceipt = await tx.wait();
+
+      const responseCode = txReceipt.events.filter(e => e.event === 'ResponseCode')[0].args.responseCode;
+      expect(responseCode).to.equal(TX_SUCCESS_CODE);
+
+      const isTokenFlag = txReceipt.events.filter(e => e.event === 'IsToken')[0].args.isToken;
+      expect(isTokenFlag).to.equal(true);
+    });
+    it('should returns 0 for getTokenType with passed HTS token', async function() {
```
Comment (Nana-EC) [inline packages/server/tests/acceptance/htsPrecompile.spec.ts:629]:
```suggestion
    it('should returns 0 for getTokenType with passed FUNGIBLE_COMMON token', async function() {
```

File: packages/server/tests/acceptance/htsPrecompile.spec.ts (hunk 1 above)
Comment (Nana-EC) [inline packages/server/tests/acceptance/htsPrecompile.spec.ts:639]:
```suggestion
    it('should returns 1 for getTokenType with passed HTS NON_FUNGIBLE_UNIQUE token', async function() {
```

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Looks good.
Some clarifying suggestions.

Once addressed please move PR to draft.
Let's wait on services tag before checking this PR in.

**Review by Nana-EC — DISMISSED** [review]
LG, please update PR description to match your changes

**Review by Ivo-Yankov — APPROVED** [review]

**Pull request merged**